src/farajaland/features/forms/register.json

```diff
@@ -166,15 +166,11 @@
                         "mapping": {
                           "mutation": {
                             "operation": "changeHirerchyMutationTransformer",
-                            "parameters": [
-                              "informant.otherRelationship"
-                            ]
+                            "parameters": ["informant.otherRelationship"]
```

Comment:
Try to avoid formatting changes like this. Makes reviewing quite difficult as it's more difficult to see where the actual changed part is 😔

Reply:
Yeah, I was also realizing that, but whenever I save code on vscode, it automatically change with standard format :(

Reply:
Yup, that definitely makes it more difficult. Hopefully it works ok for everyone from this point forward :)